frame/generate-bags/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
```

Comment:
I'm still more a fan of having `generate-bags` inside `bags-list` to minimize the footprint and keep the top directly of frame tidier.

Reply:
Forgot if I already mentioned, but the issue is that `generate-bags` needs a dependency on `pallet-staking` and which puts us in an unnecessarily tricky spot with cyclical deps. If we are worried about code org I think nesting the crate itself inside `bags-list` makes sense.